Bck 2 develop(Connection server with client)

KrayVlad opened the pull request.

KrayVlad committed:
Add files via upload

KrayVlad committed:
Delete server directory

KrayVlad committed:
Add files via upload

KrayVlad committed:
Add files via upload

Cardun reviewed (changes requested):
Please resolve messages that I left in commits, this PR doesn't have any changes stated

volkrasn committed:
Created DB interface and plugs

Created DB interface. It works like plugs. You can pass to method error code and method will return it

volkrasn committed:
Added DB Errors. Changed errors in DB interface

KrayVlad committed:
Resolved variable name

KrayVlad committed:
Merge remote-tracking branch 'origin/BCK-2_develop' into BCK-2_develop

KrayVlad committed:
Moved from server directory

KrayVlad commented:
Resolved variable name

Cardun commented:
Closing, because everything was moved to BCK-6 branch
